app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpCreationViewModel.kt

```diff
@@ -136,6 +165,12 @@ class MeetUpCreationViewModel @Inject constructor(
         )
         if (uuid == null) {
             // create new meetup
+            // Make sure the meetup start at least now when it is created
+            if (startDate.value!!.isBefore(Calendar.getInstance())) {
```

Comment:
Is it sure that startDate.value is not null?

Reply:
yes, it is set at line 26.